packages/app/popup.html

```diff
@@ -11,7 +11,7 @@
       body {
         height: 600px;
         width: 350px;
-        overflow: hidden;
+        overflow: overlay;
```

Comment:
pretty subtle change but can have a lot of impact. did you test this in windows ?

I was reading and:

> On the other hand, with overflow: overlay, a scrollable overlay is created regardless of whether the content fits within the defined dimensions of the element or not. This means that even if the content does not exceed the dimensions of the element, a scrollbar will still be created, and the content will be partially hidden under the overlay.
> So, in summary, overflow: auto only creates a scrollbar if necessary, while overflow: overlay always creates a scrollable overlay, regardless of whether the content fits within the defined dimensions or not.


Looks like will always show a scrollbar, but in mac:
- will not show always 
- shows a cool scrollbar

but in windows probably:
- will show always
- will show that ugly  default scrollbar, which we normally use `coreStyles.coreStyles()` to have a cool scrollbar always instead.